crates/precompile/src/bls12_381/g1_msm.rs

```diff
@@ -39,7 +33,7 @@ pub(super) fn g1_msm(input: &Bytes, gas_limit: u64) -> PrecompileResult {
         return Err(PrecompileError::OutOfGas);
     }
 
-    let mut g1_points: Vec<blst_p1_affine> = Vec::with_capacity(k);
```

Comment:
This type can be inferred, removed it so we don't need to explicitly import blst